README.md

```diff
@@ -4,7 +4,7 @@
 
 `random-words` generates random words for use as sample text. We use it to generate random blog posts when testing [Apostrophe](http://apostrophecms.org).
 
-Cryptographic-quality randomness is NOT the goal, as speed matters for generating sample text and security does not. `Math.random()` is used.
+Cryptographic-quality randomness is NOT the goal, as speed matters for generating sample text and security does not. As such, `Math.random()` is used in most cases. Alternatively, the [seedrandom](https://www.npmjs.com/package/seedrandom) package is used when the `seed` option is used.
```

Comment:
Can you please add an extra paragraph describing what the `seed` option does and how it is used.